Automattic/wp-calypso
# Editing Toolkit: Load patterns from the rest API endpoint

Remove block patterns from within the codebase, and load them via the new rest API endpoint.

#### Testing instructions

* Load the editor and the patterns inserter and confirm you see all the following categories:

<img width="367" alt="Screen Shot 2020-09-24 at 2 36 02 PM" src="https://user-images.githubusercontent.com/1464705/94202886-5476fe00-fe73-11ea-9b34-15dc0af37452.png">

* Try inserting a pattern and confirm this works as expected.
* Switch your blog to a different language (settings->general)
* Confirm that you can still see patterns in the pattern inserter
* Some patterns may be translated (still confirming this).
